src/parse-mockdb.js

```diff
@@ -619,9 +635,12 @@ function handleGetRequest(request) {
       match = _.omit(match, toOmit);
     }
 
+    if (data.keys) {
+      match = projectFields([match], data.keys)[0];
+    }
+
```

Comment:
Don't you want those four default fields (ACL, objectId, createdAt, and updatedAt) even if `data.keys` is not provided?

Reply:
If data.keys isn't provided, we want to return all the fields, so we skip this projection operation entirely 😃

Reply:
Oh, wait, were they always there? And you're just including them up above so that `_.pick()` doesn't remove them?

Reply:
Oh, got it.